### ISSUE TITLE
add arrows to the end of edges under map view

### ISSUE THREAD

**Issue by ungentilgarcon** [topogram/topogram#21]
this would be cool

**Comment by ungentilgarcon** [comment]
Ok, we need this back,can https://github.com/bbecquet/Leaflet.PolylineDecorator do the job?

**Comment by clemsos** [comment]
Yes looks great. Should be able to add it here https://github.com/topogram/topogram/blob/master/imports/client/ui/components/geoMap/GeoEdges.jsx <https://github.com/topogram/topogram/blob/master/imports/client/ui/components/geoMap/GeoEdges.jsx>
